wandb/data_types.py

```diff
@@ -236,6 +236,7 @@ def get_user_name_length(index, row): return {"feature_01": len(row["user"])}
 
     MAX_ROWS = 10000
     MAX_ARTIFACT_ROWS = 200000
+    ENFORCE_MAX_ROW_LIMIT = False
```

Comment:
@dmitryduev should we be doing this with settings? is there a way to?

Reply:
great idea to do it that way, I'll explain.